### PR TITLE
[INLONG-10377][Dashboard] Add Source Data Field Template

### LINKED ISSUE #10377: [Improve][DashBoard] add Source Data Field Template
### Description

[Improve][DashBoard] add Source Data Field Template

### InLong Component

InLong Dashboard

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by wohainilaodou**
Fixes #10377 

### Motivation
Add template functionality to metadata fields

### Modifications

A source data template page has been created, which includes the addition, modification, and query functions. On the data flow editing page, templates can be selected to fill in the source data fields

### Verifying this change
image
image

image

### CODE REVIEW

File: inlong-dashboard/src/plugins/streams/common/StreamDefaultInfo.ts [hunk 1]
```diff
@@ -222,6 +222,37 @@ export class StreamDefaultInfo implements DataWithBackend, RenderRow, RenderList
...
+          method: 'POST',
+          data: {
+            keyword,
+            pageNum: 1,
+            pageSize: 20,
+          },
+        }),
+        requestParams: {
+          formatResult: result => {
+            console.log('result.log', result);
```
Comment (leezng) [inline inlong-dashboard/src/plugins/streams/common/StreamDefaultInfo.ts:242]: remove console

File: inlong-dashboard/src/ui/pages/Clusters/NodeEditModal.tsx [hunk 2]
```diff
@@ -79,13 +79,43 @@ const NodeEditModal: React.FC<NodeEditModalProps> = ({ id, type, clusterId, ...m
...
+      manual: true,
+      onSuccess: result => {
+        let temp = result?.list
+          ?.filter(item => item.type === 'INSTALLER')
+          .map(item => ({
+            ...item,
+            label: `${item.name} ${item.version}`,
+            value: item.id,
+          }));
+        console.log('result', temp);
```
Comment (leezng) [inline inlong-dashboard/src/ui/pages/Clusters/NodeEditModal.tsx:101]: remove console

File: inlong-dashboard/src/ui/pages/Clusters/NodeEditModal.tsx [hunk 3]
```diff
@@ -79,13 +79,43 @@ const NodeEditModal: React.FC<NodeEditModalProps> = ({ id, type, clusterId, ...m
...
       // open
       form.resetFields();
       if (id) {
         getData(id);
+      } else {
+        if (type === 'AGENT') {
+          getAgentInstall();
+        }
       }
+      console.log(form.getFieldValue('installer'));
```
Comment (leezng) [inline inlong-dashboard/src/ui/pages/Clusters/NodeEditModal.tsx:118]: rm

File: inlong-dashboard/src/ui/pages/Clusters/NodeEditModal.tsx [hunk 4]
```diff
@@ -79,13 +79,43 @@ const NodeEditModal: React.FC<NodeEditModalProps> = ({ id, type, clusterId, ...m
...
+      method: 'POST',
+      data: {
+        pageNum: 1,
+        pageSize: 9999,
+      },
+    }),
+    {
+      manual: true,
+      onSuccess: result => {
+        let temp = result?.list
```
Comment (leezng) [inline inlong-dashboard/src/ui/pages/Clusters/NodeEditModal.tsx:94]: let -> const

**Review by leezng — COMMENTED** [review]
Too many console, please check it.

File: inlong-dashboard/src/ui/pages/Clusters/NodeEditModal.tsx (hunk 4 above)
Comment (wohainilaodou) [reply inlong-dashboard/src/ui/pages/Clusters/NodeEditModal.tsx:94]: done

File: inlong-dashboard/src/ui/pages/Clusters/NodeEditModal.tsx (hunk 3 above)
Comment (wohainilaodou) [reply inlong-dashboard/src/ui/pages/Clusters/NodeEditModal.tsx:118]: done

File: inlong-dashboard/src/ui/pages/Clusters/NodeEditModal.tsx (hunk 2 above)
Comment (wohainilaodou) [reply inlong-dashboard/src/ui/pages/Clusters/NodeEditModal.tsx:101]: done

File: inlong-dashboard/src/plugins/streams/common/StreamDefaultInfo.ts (hunk 1 above)
Comment (wohainilaodou) [reply inlong-dashboard/src/plugins/streams/common/StreamDefaultInfo.ts:242]: done

**Review by haifxu — APPROVED** [review]

**Review by leezng — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**